travel-2018.html

```diff
@@ -0,0 +1,236 @@
+  <link href="custom/css/buttons.css" rel="stylesheet">
+  <link rel="shortcut icon" href="img/favicon.png">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat+Alternates:300,400,500,600" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+</head>
+
+
+<body id="page-top" data-spy="scroll" data-target=".navbar-custom">
+  <!-- Preloader -->
+  <!-- <div id="preloader">
+    <div id="load"></div>
+  </div> -->
+
+  <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+
+      <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
+        <img src="./img/banner-logo.png" class="logo" alt="banner-logo" style="width: 150px; display: inline-block;">
+      </a>
+
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"
+            aria-expanded="false">
+          <span class="sr-only">Toggle navigation</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+      </div>
+
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav nav-justified">
+          <li><a href="index.html#home">Home</a></li>
+          <li><a href="index.html#journey">Journey</a></li>
+          <li><a href="index.html#speakers">Speakers</a></li>
+          <!-- <li><a href="#conference">Schedule</a></li> -->
+          <li><a href="index.html#sponsors">Sponsors</a></li>
+          <li><a href="index.html#tickets">Tickets</a></li>
+          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
+          <li><a href="index.html#venue">Venue</a></li>
+          <li><a href="faq.html">FAQ</a></li>
+          <li><a href="volunteer.html">Volunteer</a></li>
+          <li class="active"><a href="travel-2018.html">Travel</a></li>
+          <li><a href="coc.html">COC</a></li>
+        </ul>
+      </div>
+
+  </nav>
+
+
+  <!-- Section: about -->
+
+  <section id="faq" class="home-section splitsection" style="margin-top:0">
+    <div class="heading-about">
+      <div class="container">
+        <h2 >How to get to PyCon</h2>
+        <div class="pycon-travel">
+        	<p><u><a href="https://wikitravel.org/en/Hyderabad">Here is a handy Wiki page</a></u> describing about the various ways you can travel to Hyderabad. The main points have been highlighted below.</p>
+       		<ul>
+       			<li>
+              <h3>Via Air</h3>
+       				<p><b>To Hyderabad: </b>Domestic connectivity to Hyderabad is excellent with Indian airlines operating from here including Air India, Air India Express, Indian Airlines, Indigo Airlines, Jet Airways, JetLite, Trujet and SpiceJet. (Kingfisher is now defunct, and Trujet is new).</p>
```

Comment:
Can you add information about international connectivity, or if there are more than one terminals. Which gate would be the most feasible to exit out of.

Reply:
RGIA has combined international and domestic airports. Apart from that I could not find any information on which would be the most appropriate gate to exit out of.